packages/cli/generators/controller/index.js

```diff
@@ -127,6 +127,16 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
             default: modelList[0],
             validate: utils.validateClassName,
           },
+          {
+            type: 'input',
+            name: 'modelNamePlural',
+            message:
+              'What is the custom plural form of the model (for building REST URL)',
+            when: this.artifactInfo.modelName === undefined,
+            default: response =>
+              utils.kebabCase(utils.pluralize(response.modelName)),
+            validate: utils.validateUrlSlug,
```

Comment:
The prompt would look something like this:
```shell
What is the name of the model to use with this CRUD repository? Customer
What is the plural form of the model? (customers)
```
When given a bad input (foo#bar):
```shell
What is the plural form of the model? (customers)
Invalid url slug. Suggested slug: foo-bar
```

Reply:
Once we rephrase the question to ask for URL path, should we include the leading `/` in the default value?

If we do so, I think we should still accept and normalize values without a leading slash. For example:

 - User enters "foo-bar" -> we use `/foo-bar`
 - User enters "foo/" -> we use `/foo`